### PR TITLE
rename quick-install-guide.md

### PR DESCRIPTION
**Pull request by sunzhaoyang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [x] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

File: data-migration/aurora/increment-aurora.md [hunk 1]
```diff
@@ -19,29 +19,23 @@ This document describes how to use DM for incremental replication.
 
 DM relies on the `ROW`-formatted binlog for incremental replication, so you need to enable the binary logging for an Aurora MySQL cluster. For the configuration instructions, refer to Amazon's document [Enable binary for an Aurora Cluster](https://aws.amazon.com/premiumsupport/knowledge-center/enable-binary-logging-aurora/?nc1=h_ls).
 
-If global transaction identifiers (GTIDs) are enabled in Aurora, you can migrate data based on GTIDs. For how to enable it, refer to Amazon's document [Configuring GTID-Based Replication for an Aurora MySQL Cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/mysql-replication-gtid.html#mysql-replication-gtid.configuring-aurora).
-
-> **Note:**
->
-> + GTID-based data migration requires MySQL 5.7 (Aurora 2.04) version or later.
+It is recommmend to  migrate data based on GTIDs (global transaction identifiers). For how to enable it, refer to Amazon's document [Configuring GTID-Based Replication for an Aurora MySQL Cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/mysql-replication-gtid.html#mysql-replication-gtid.configuring-aurora).
```
Comment (hfxsd) [inline data-migration/aurora/increment-aurora.md:22]:
```suggestion
It is recommmend to  migrate data based on GTIDs (global transaction identifiers). For more information about how to enable it, refer to Amazon's document [Configuring GTID-Based Replication for an Aurora MySQL Cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/mysql-replication-gtid.html#mysql-replication-gtid.configuring-aurora).
```

File: data-migration/aurora/increment-aurora.md [hunk 2]
```diff
@@ -51,6 +45,15 @@ from:
...
 ```
 
+| Parameter | Description |
+| :--------| :------------|
+| `--master-addr`                     | Master API server addr. |
+| `operate-source`                    | `create`/`update`/`stop`/`show` upstream MySQL/MariaDB source. |
+
+Usage of `dmctl encrypt`: [Encrypt the database password using dmctl](https://docs.pingcap.com/tidb-data-migration/stable/manage-source#encrypt-the-database-password)
+
+All avaiable command of `dmctl` could be found in [Maintain DM Clusters Using dmctl](https://docs.pingcap.com/tidb-data-migration/stable/dmctl-introduction)
```
Comment (hfxsd) [inline data-migration/aurora/increment-aurora.md:55]:
```suggestion
You can find all available `dmctl` commands in [Maintain DM Clusters Using dmctl](https://docs.pingcap.com/tidb-data-migration/stable/dmctl-introduction)
```

File: data-migration/aurora/increment-aurora.md [hunk 3]
```diff
@@ -51,6 +45,15 @@ from:
...
+
 When the data sources are successfully added, the return information of each data source includes a DM-worker bound to it.
```
Comment (hfxsd) [inline data-migration/aurora/increment-aurora.md:57]:
```suggestion
When the data sources are successfully added, each data source returns information that includes a DM-worker bound to it.
```

**Review by hfxsd — APPROVED** [review]

File: data-migration/aurora/from-snapshot.md [hunk 4]
```diff
@@ -9,8 +9,8 @@ This document describes how to migrate full data from Amazon Aurora MySQL to TiD
 
 ## Prerequisites
 
-- [Deploy TiDB Lighting using TiUP](/quick_install_tools.md)
-- [Deploy Dumping using TiUP](/quick_install_tools.md)
+- [Deploy Lighting using TiUP](/quick-install-tools.md)
```
Comment (Liuxiaozhen12) [inline data-migration/aurora/from-snapshot.md:12]:
```suggestion
- [Deploy Lightning using TiUP](/quick-install-tools.md)
```

File: data-migration/aurora/from-snapshot.md [hunk 5]
```diff
@@ -101,6 +100,14 @@ tiup tidb-lightning -config tidb-lightning.toml -d ./schema -no-schema=false
 
 In this example, TiDB Lightning is only used to create table schemas, so the above command runs very fast. In average, It only takes one second to execute ten table creation statements.
 
+| Parameter | Description |
+| :--------| :------------|
+| `-config`                     | Reads global configuration from file. If not specified, the default value is used. |
+| `-d`                          | Directory or [external storage URL](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages) of the data dump to read from |
+| `-no-schema`                  | Ignore schema files, and get schema directly from TiDB |
+
+You can find more parameters in [TiDB Lightning Configuration](https://docs.pingcap.com/tidb/stable/tidb-lightning-configuration)
```
Comment (Liuxiaozhen12) [inline data-migration/aurora/from-snapshot.md:109]:
```suggestion
For more parameters, refer to [TiDB Lightning Configuration](https://docs.pingcap.com/tidb/stable/tidb-lightning-configuration).
```

File: data-migration/aurora/increment-aurora.md [hunk 6]
```diff
@@ -19,29 +19,23 @@ This document describes how to use DM for incremental replication.
 
 DM relies on the `ROW`-formatted binlog for incremental replication, so you need to enable the binary logging for an Aurora MySQL cluster. For the configuration instructions, refer to Amazon's document [Enable binary for an Aurora Cluster](https://aws.amazon.com/premiumsupport/knowledge-center/enable-binary-logging-aurora/?nc1=h_ls).
 
-If global transaction identifiers (GTIDs) are enabled in Aurora, you can migrate data based on GTIDs. For how to enable it, refer to Amazon's document [Configuring GTID-Based Replication for an Aurora MySQL Cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/mysql-replication-gtid.html#mysql-replication-gtid.configuring-aurora).
-
-> **Note:**
->
-> + GTID-based data migration requires MySQL 5.7 (Aurora 2.04) version or later.
+It is recommmend to  migrate data based on GTIDs (global transaction identifiers). For more information about how to enable it, refer to Amazon's document [Configuring GTID-Based Replication for an Aurora MySQL Cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/mysql-replication-gtid.html#mysql-replication-gtid.configuring-aurora).
```
Comment (Liuxiaozhen12) [inline data-migration/aurora/increment-aurora.md:22]:
```suggestion
It is recommended to migrate data based on GTIDs (global transaction identifiers). For more information about how to enable it, refer to Amazon's document [Configuring GTID-Based Replication for an Aurora MySQL Cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/mysql-replication-gtid.html#mysql-replication-gtid.configuring-aurora).
```

File: data-migration/aurora/increment-aurora.md [hunk 7]
```diff
@@ -19,29 +19,23 @@ This document describes how to use DM for incremental replication.
...
 
 # To migrate data based on GTID, you need to set this item to true.
 enable-gtid: false
 
 from:
-  host: "test-dm-2-0.cluster-czrtqco96yc6.us-east-2.rds.amazonaws.com"
-  user: "root"
-  password: "OiG90CGm3CEbXan6ZSd/SUAsofxJAZo="  # It is recommended to use database passwords encrypted by `dmctl encrypt`,see [Encrypt the database password using dmctl](manage-source.md#encrypt-the-database-password)
+  host: "${host}" # address of database, eg: test-dm-2-0.cluster-czrtqco96yc6.us-east-2.rds.amazonaws.com
+  user: "${user_name}"    # username of database, eg: root
```
Comment (Liuxiaozhen12) [inline data-migration/aurora/increment-aurora.md:37]:
```suggestion
  user: "${user_name}"    # username of database, for example: root
```

File: data-migration/aurora/increment-aurora.md [hunk 8]
```diff
@@ -19,29 +19,23 @@ This document describes how to use DM for incremental replication.
...
+  password: "${password}"  # It is recommended to use database passwords encrypted by `dmctl encrypt`.
```
Comment (Liuxiaozhen12) [inline data-migration/aurora/increment-aurora.md:38]:
```suggestion
  password: "${password}"  # It is recommended to use database passwords encrypted by `dmctl encrypt`
```
配置说明里一般不带句号

File: data-migration/aurora/increment-aurora.md [hunk 9]
```diff
@@ -51,7 +45,16 @@ from:
 tiup dmctl --master-addr 127.0.0.1:8261 operate-source create dm-test/source1.yaml
 ```
 
-When the data sources are successfully added, the return information of each data source includes a DM-worker bound to it.
+| Parameter | Description |
+| :--------| :------------|
+| `--master-addr`                     | Master API server addr. |
```
Comment (Liuxiaozhen12) [inline data-migration/aurora/increment-aurora.md:50]:
```suggestion
| `--master-addr`                     | Master API server address. |
```

File: data-migration/aurora/increment-aurora.md [hunk 10]
```diff
@@ -51,7 +45,16 @@ from:
...
+| `operate-source`                    | `create`/`update`/`stop`/`show` upstream MySQL/MariaDB source. |
+
+Usage of `dmctl encrypt`: [Encrypt the database password using dmctl](https://docs.pingcap.com/tidb-data-migration/stable/manage-source#encrypt-the-database-password)
+
+You can find all available `dmctl` commands in [Maintain DM Clusters Using dmctl](https://docs.pingcap.com/tidb-data-migration/stable/dmctl-introduction)
```
Comment (Liuxiaozhen12) [inline data-migration/aurora/increment-aurora.md:55]:
```suggestion
You can find all the available `dmctl` commands in [Maintain DM Clusters Using dmctl](https://docs.pingcap.com/tidb-data-migration/stable/dmctl-introduction).
```

File: data-migration/aurora/from-snapshot.md [hunk 11]
```diff
@@ -52,19 +52,22 @@ vim tidb-lighting.toml
...
-port = ${port}
-user = "${user_name}"
-password = "${password}" 
-# The default PD address of the cluster.
-pd-addr = "127.0.0.1:2379"
+host = "${host}" # target database address ,eg: 172.16.128.1
+port = ${port}   # target database port, eg: 4000
+user = "${user_name}"  # target database username, eg: root
+password = "${password}"  # target database password
+pd-addr = "${pd_address}"  # The default PD address of the cluster. eg: 127.0.0.1:2379
```
Comment (Liuxiaozhen12) [inline data-migration/aurora/from-snapshot.md:63]:
```suggestion
host = "${host}" # the target database address, for example: 172.16.128.1
port = ${port}   # the target database port, for example: 4000
user = "${user_name}"  # the target database username, for example: root
password = "${password}"  # the target database password
pd-addr = "${pd_address}"  # The default PD address of the cluster, for example: 127.0.0.1:2379
```

File: data-migration/aurora/from-snapshot.md [hunk 12]
```diff
@@ -52,19 +52,22 @@ vim tidb-lighting.toml
...
 
 [tikv-importer]
-# Uses Local-backend for best performance. You can also choose TiDB-backend or Importer-backend according to your need. For detailed introduction of the three backend modes, see [TiDB Lightning Backends](/tidb-lightning/tidb-lightning-backends.md).
+
+# The "Local" backend mode is used by default for best performance, which is suitable for large data volumes above TB level. But in this mode, the downstream TiDB cannot provide services during the import period.
```
Comment (Liuxiaozhen12) [inline data-migration/aurora/from-snapshot.md:67]:
```suggestion
# The "Local" backend mode is used by default for the best performance, which is suitable for large data volumes larger than 1 TiB. But in this mode, the downstream TiDB cannot provide services during the import process.
```

File: data-migration/aurora/from-snapshot.md [hunk 13]
```diff
@@ -52,19 +52,22 @@ vim tidb-lighting.toml
...
+# The data volume below TB level can also adopt the "tidb" backend mode, and downstream TiDB can provide services normally. For more information about the backend mode, please refer to: https://docs.pingcap.com/tidb/stable/tidb-lightning-backends
```
Comment (Liuxiaozhen12) [inline data-migration/aurora/from-snapshot.md:68]:
```suggestion
# The data volume less than 1 TiB can also adopt the "tidb" backend mode, and downstream TiDB can provide services normally. For more information about the backend mode, refer to [TiDB Lightning](/tidb-lightning/tidb-lightning-backends.md).
```

File: data-migration/aurora/from-snapshot.md [hunk 14]
```diff
@@ -83,11 +86,7 @@ table = '$2'
 type = '$3'
 ```
 
-> **Note:**
->
-> - If TLS is enabled in the target TiDB cluster, you also need to configure TLS.
-
-For more configurations, refer to [TiDB Lightning Configuration](/tidb-lightning/tidb-lightning-configuration.md).
+Refer to the [TiDB Lightning Configuration](/tidb-lightning/tidb-lightning-configuration.md) if you want to configure TLS in the target TiDB cluster, or for more configurations.
```
Comment (Liuxiaozhen12) [inline data-migration/aurora/from-snapshot.md:89]:
```suggestion
If you want to configure TLS in the target TiDB cluster, or know about more configurations, refer to the [TiDB Lightning Configuration](/tidb-lightning/tidb-lightning-configuration.md).
```

**Pull request merged**